.github/workflows/tests.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-latest, macos-latest]
-        pyv: ["3.8", "3.9", "3.10"]
+        pyv: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
3.11, 3.12 could be added as a separate PR if they require more changes

Reply:
I'll wait for the CI. Seemed fine locally.